src/resources/chat/completions.ts

```diff
@@ -205,19 +359,24 @@ export interface ChatCompletionMessage {
   /**
    * The role of the author of this message.
    */
-  role: ChatCompletionRole;
+  role: 'assistant';
```

Comment:
I'm not sure if this will be noticed, but I'm curious why this was changed to just 'assistant' from ChatCompletionRole? It broke our TS build that was using ChatCompletionMessage.

I'll review changing to ChatCompletionMessageParam as the type to use before we call openai.chat.completions.create